packages/core/src/env-set/create-pool-by-env.ts

```diff
@@ -24,18 +73,28 @@ const createPoolByEnv = async (isTest: boolean) => {
       throw error;
     }
 
-    const answer = await inquirer.prompt({
-      name: 'dsn',
-      message: `No Postgres DSN (${key}) found in env variables. Please input the DSN which points to Logto database:`,
-    });
+    const [dsn, needsSeed] = await inquireForLogtoDsn(key);
 
-    if (!answer.dsn) {
+    if (!dsn) {
       throw error;
     }
 
-    appendDotEnv(key, answer.dsn);
+    const cli = createDatabaseCli(dsn);
```

Comment:
nit: Use object destructuring 
```
    const { createTables, seedTables, pool } = createDatabaseCli(dsn);
```

Reply:
ah it's by design to reduce naming pollution